lighthouse-core/audits/metrics/total-blocking-time.js

```diff
@@ -14,7 +14,7 @@ const UIStrings = {
   title: 'Total Blocking Time',
   /** Description of the Total Blocking Time (TBT) metric, which calculates the total duration of blocking time for a web page. Blocking times are time periods when the page would be blocked (prevented) from responding to user input (clicks, taps, and keypresses will feel slow to respond). This is displayed within a tooltip when the user hovers on the metric name to see more. No character length limits.*/
   description: 'Sum of all time periods between FCP and Time to Interactive, ' +
-      'when task length exceeded 50ms, expressed in milliseconds.',
+      'when task length exceeded 50ms, expressed in milliseconds. [Learn more](https://web.dev/lighthouse-total-blocking-time).',
```

Comment:
Article should be live soon

Reply:
@kaycebasques is it likely that future articles will start having a `lighthouse-` prefix in their slug?

Reply:
Yes. Philip is writing canonical docs for each metric <sup>1</sup> and it seems like those docs should get the high-profile URLs.

I would prefer the Lighthouse docs to go under `web.dev/lighthouse/total-blocking-time` (for example) but it seems like it may be a fair chunk of work for a minor cosmetic gain. cc @robdodson 

<sup>1</sup> https://github.com/GoogleChrome/web.dev/pull/1639#issuecomment-540241698 explains why I think we should have Lighthouse docs for each metric in addition to Philip's canonical docs

Reply:
> I would prefer the Lighthouse docs to go under web.dev/lighthouse/total-blocking-time (for example) but it seems like it may be a fair chunk of work for a minor cosmetic gain. cc @robdodson

yeah we've avoided creating directories in our URLs because content is often in flux. Something might start off as a blog and then need to migrate to a section in /learn (which is what will happen with Phil's [LCP](https://web.dev/largest-contentful-paint/). So, for the moment, we're using the same approach as css-tricks for their [guides section](https://css-tricks.com/guides/). Everything hangs off of the root, and landing pages just collect those links.